### ISSUE TITLE
Herding issues

### ISSUE THREAD

**Issue by kevinSuttle** [npm/enterprise-configurator#41]
I feel like these are all related, specifically to npme.
https://github.com/npm/npme-installer/issues/122
https://github.com/npm/npm/issues/12008
https://github.com/npm/npm/issues/11137
https://github.com/npm/npm/issues/10782
https://github.com/npm/npm/issues/10835

**Comment by bcoe** [comment]
@kevinSuttle we've actually been moving away from using this configuration library in production, and in npm Enterprise; it was one of the first libraries I wrote here, and it has not aged well.

npm Enterprise is now using a 12-factor-app approach to configuration: http://12factor.net/

And we're using the library yargs for most of our configuration needs: https://github.com/yargs/yargs

The CLI uses a separate approach for managing configuration.

tldr; we should probably put a deprecation notice on this library, to reduce confusion.
